hm-post-repeat.php

```diff
@@ -248,6 +262,55 @@ function repeating_post_types() {
 }
 
 /**
+ * All available repeat schedules.
+ *
+ * @return array An array of all available repeat schedules
+ */
+function repeating_schedules() {
+
+	/**
+	 * Enable support for additional schedules.
+	 *
+	 * @param array[] $schedules Schedule array items.
+	 */
+	$schedules = apply_filters( 'hm_post_repeat_schedules', array(
+		'daily'   => array( 'interval' => '1 day',   'display' => __( 'Daily',   'hm-post-repeat' ) ),
+		'weekly'  => array( 'interval' => '1 week',  'display' => __( 'Weekly',  'hm-post-repeat' ) ),
+		'monthly' => array( 'interval' => '1 month', 'display' => __( 'Monthly', 'hm-post-repeat' ) ),
+	) );
+
+	foreach ( $schedules as $slug => &$schedule ) {
+		$schedule['slug'] = $slug;
+	}
+
+	return $schedules;
+
+}
+
+/**
+ * Get the repeating schedule of the given post_id.
+ *
+ * @param int $post_id The id of the post you want to check.
+ * @return array|null The schedule to repeat by, or null if invalid.
+ */
+function get_repeating_schedule( $post_id ) {
+
+	if ( ! is_repeating_post( $post_id ) ) {
+		return null;
```

Comment:
For consistencies sake let's just `return;` here rather than explicitly returning `null`